<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QDialog" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>353</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>19</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_07">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_08">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_01">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_05">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_02">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>109</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_03">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_04">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-right-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_06">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-top-width: 2px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_09">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>108</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0px;
padding: 0px;
border-style: solid;
border-color: black;
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="text">
        <string>&amp;New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quit">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionButton01">
   <property name="text">
    <string>button01</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionButton02">
   <property name="text">
    <string>button02</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionButton03">
   <property name="text">
    <string>button03</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionButton04">
   <property name="text">
    <string>button04</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionButton05">
   <property name="text">
    <string>button05</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionButton06">
   <property name="text">
    <string>button06</string>
   </property>
   <property name="toolTip">
    <string>button06</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionButton07">
   <property name="text">
    <string>button07</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="actionButton08">
   <property name="text">
    <string>button08</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="actionButton09">
   <property name="text">
    <string>button09</string>
   </property>
   <property name="toolTip">
    <string>button09</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>GameBoard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton01</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_01</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton02</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_02</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton03</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_03</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton04</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_04</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton05</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_05</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton06</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_06</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton07</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_07</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton08</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_08</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton09</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_09</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
